android/app/src/main/java/org/mozilla/magnet/database/HistoryDatabase.java

```diff
@@ -0,0 +1,9 @@
+package org.mozilla.magnet.database;
+
+import java.util.Date;
+
+interface HistoryDatabase {
```

Comment:
It does work cause it has default visibility. I don't know if you were intended to do that, but that interface will be available just to the classes that are in that package: http://docs.oracle.com/javase/tutorial/java/javaOO/accesscontrol.html

If that's your intention, then leave as it is :)